File: dbmigrator_back/src/main/java/com/seguridata/tools/dbmigrator/data/model/ExecutionStatisticsModel.java
package com.seguridata.tools.dbmigrator.data.model;

import com.seguridata.tools.dbmigrator.data.constant.ExecutionResult;
import com.seguridata.tools.dbmigrator.data.constant.ExecutionStatus;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;

@Getter @Setter
public class ExecutionStatisticsModel implements Serializable {
	private static final long serialVersionUID = 3210797572383776006L;

	private String planId;
    private String planName;
    private ExecutionResult result;
    private ExecutionStatus status;
    private Double progress;
    private Long rowsProcessed;
    private Long rowsForCompletion;
}

File: seguridata-migrator/src/app/common/models/execution-statistics-model.ts
import { ExecutionResult } from "../enums/execution-result";
import { ExecutionStatus } from "../enums/execution-status";

export interface ExecutionStatisticsModel {
  planId: string;
  planName: string;
  result: ExecutionResult;
  status: ExecutionStatus;
  progress: number;
}
